lerobot-web-fe/src/App.tsx

```diff
@@ -1,5 +1,20 @@
+import { useEffect } from 'react';
 import AppRouter from './routers/AppRouter';
+import { useStatusStore } from './stores/status.store';
+import StatusPoller from './components/StatusPoller';
 
 export default function App() {
-  return <AppRouter />;
+  useEffect(() => {
+    const { apiUrl, setApiUrl } = useStatusStore.getState();
```

Comment:
Instead of destruct, please use the separated selector-based solution here as well.

```suggestion
  const apiUrl = useStatusStore((s) => s.apiUrl);
  const setApiUrl = useStatusStore((s) => s.setApiUrl);

```

Reply:
It will cause an invalid hook call. The useStatusStore(s => s.apiUrl) is for reactive, efficient updates. The  useStatusStore.getState().apiUrl usage is safe, static, and meets React's hook rules.